project/src/pages/offers-list/offers-list.tsx

```diff
@@ -0,0 +1,22 @@
+import { Offers } from '../../types/offer';
+import OfferCard from '../../components/card';
+
+type OffersListProps = {
+  offerCount: number;
+  offers: Offers;
+}
+
+function OffersList(props: OffersListProps): JSX.Element {
+  const {offers} = props;
+  const {offerCount} = props;
```

Comment:
лучше сразу деструктурируй 
```suggestion
function OffersList({ offerCount, offers }: OffersListProps): JSX.Element {
```